src/lib/mina.ts

```diff
@@ -685,61 +687,64 @@ function LocalBlockchain({
 // assert type compatibility without preventing LocalBlockchain to return additional properties / methods
 LocalBlockchain satisfies (...args: any) => Mina;
 
+type MinaNetworkEndpoints = {
+  mina: string | string[];
+  archive?: string | string[];
+  lightnetAccountManager?: string;
+};
+
 /**
  * Represents the Mina blockchain running on a real network
  */
 function Network(graphqlEndpoint: string): Mina;
-function Network(endpoints: {
-  mina: string | string[];
-  archive?: string | string[];
-  lightnetAccountManager?: string;
-}): Mina;
+function Network(endpoints: MinaNetworkEndpoints): Mina;
```

Comment:
please revert this change - it would lead to worse intellisense

also, it makes more sense to me to add `networkId` as a property of the same object than to make it a nested object

Reply:
> please revert this change - it would lead to worse intellisense

Will do, thanks.

> also, it makes more sense to me to add `networkId` as a property of the same object than to make it a nested object

I also was thinking about it but since there might be more options in future that probably won't be related to endpoints I decided to extract it into nested object. Will refactor then.